src/main/java/me/ruibin/leto/tasklist/TaskList.java

```diff
@@ -0,0 +1,172 @@
+package me.ruibin.leto.tasklist;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import me.ruibin.leto.storage.Storage;
+import me.ruibin.leto.ui.Ui;
+
+/** Class representing the list of tasks in Leto. */
+public class TaskList {
+    private static final ArrayList<Task> list = new ArrayList<>(100);
+
+    public TaskList() {}
+
+    /** Static method to initialise the task list from csv file. */
+    public static void initFromFile() {
```

Comment:
Since you used csv to refer to the file in other methods, maybe initFromCsv would be more informative